presto-main/src/main/resources/webapp/src/components/QueryList.jsx

```diff
@@ -42,6 +42,34 @@ function getHumanReadableStateFromInfo(query) {
     );
 }
 
+
+function ResourceGroupLinks({groupId, length=35}) {
+    if (groupId && groupId.length > 0) {
+        var previousLen = 0;
```

Comment:
Also, you can use `let` instead of `var` to take advantage of its [TDZ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#temporal_dead_zone_tdz).